config/meltor.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+use Illuminate\Support\Facades\DB;
+
+return [
+
+    // Names of the Laravel connections, used as the base for the new, generated migration.
+    'connection' => [
+
+        // Connection to read the tables.
+        'data'   => fn() => DB::getDefaultConnection(),
+
+        // You may need to create a new DB connection that points to the information_schema. See README.md.
+        'schema' => 'information_schema_mysql',
```

Comment:
Why doesn't the package do that for me?

Reply:
yes i should do that

Reply:
the package now automatically creates this connection from the default DB connection config. This can also be configured to read credentials from another DB connection config.